Add a MySQL JDBC driver extension (#4067)
quarkusio/quarkus: integration-tests/jpa-mysql/README.md

> # JPA example with MySQL DB

Reviewer: ```suggestion
# JPA example with MySQL
```

## Running the tests

By default, the tests of this module are disabled.

To run the tests in a standard JVM with MySQL started as a Docker container, you can run the following command:

```
mvn clean install -Dtest-mysql -Ddocker
```

> Please note that waiting on the availability of MySQL DB port does not work on macOS.

Reviewer: ```suggestion
Please note that waiting on the availability of the MySQ port does not work on macOS.
```

This module does not work with `-Ddocker` option on this operating system.

Additionally, you can generate a native image and run the tests for this native image by adding `-Dnative`:

```
mvn clean install -Dtest-mysql -Ddocker -Dnative
```

> If you don't want to run MySQL as a Docker container, you can start your own MySQLDB server. It needs to listen on the default port and have a database called `hibernate_orm_test` accessible to the user `hibernate_orm_test` with the password `hibernate_orm_test`.

Reviewer: ```suggestion
If you don't want to run MySQL as a Docker container, you can start your own MySQL server. It needs to listen on the default port and have a database called `hibernate_orm_test` accessible to the user `hibernate_orm_test` with the password `hibernate_orm_test`.
```
